search/indexing_api.py

```diff
@@ -432,20 +432,19 @@ def update_post(doc_id, post):
     )
 
 
-def index_items(serialize_bulk_items, object_type, ids):
+def index_items(documents, object_type):
     """
     Index items based on list of item ids
 
     Args:
-        serialize_bulk_items (callable): the function to serializer a list of objects by id
+        documents (iterable of dict): An iterable with ElasticSearch documents to index
         object_type (str): the ES object type
-        ids(list of int): List of item id's
     """
     conn = get_conn()
     for alias in get_active_aliases(conn, [object_type]):
         _, errors = bulk(
             conn,
-            serialize_bulk_items(ids),
+            documents,
```

Comment:
Because this is a generator we won't be able to read from it twice, and there could be two or more `alias`es in the for loop... I'll see what we can do to work around that